app/views/home/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="row">
+  <div class="col-lg-12">
+    <div class="row">
+      <div class="col-md-4">
+        <h1>Top Movies</h1>
+        <% @movies.by_ranking.first(10).each do |movie| %>
+          <h2><%= link_to "#{movie.title}", movie_path(movie.id), action: :show %>
```

Comment:
I don't think we need `action: :show` here, as that information is already included with `movie_path(movie.id)`.